### PR TITLE
Nm dispatcher failed

### PR DESCRIPTION
**Pull request by simotek**
- Related ticket: bsc#1227081
- Needles: N/A
- Verification run: N/A

### CODE REVIEW

**Comment by simotek** [comment]
> /squash commits

Easy, I thought github was smart enough to do that.

**Comment by simotek** [comment]
> /squash commits

Done.

**Comment by simotek** [comment]
Just rebased on top of current master

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -728,5 +728,13 @@
             "sle": ["15-SP6", "15-SP7"]
         },
         "type": "ignore"
+    },
+    "boo#1227081": {
+        "description": "Activation request for 'org.freedesktop.nm_dispatcher' failed.",
```
Comment (Vogtinator) [inline data/journal_check/bug_refs.json:733]: Regex should be escaped:

```suggestion
        "description": "Activation request for 'org\.freedesktop\.nm_dispatcher' failed\.",
```

**Comment by Vogtinator** [comment]
> > /squash commits
> 
> Easy, I thought github was smart enough to do that.

It can, if the option is selected for merging.

**Comment by pevik** [comment]
> * bsc#122708

https://bugzilla.suse.com/show_bug.cgi?id=122708 has `You must enter a valid bug number!`. Maybe typo in the number.
``

File: data/journal_check/bug_refs.json (hunk 1 above)
Comment (simotek) [reply data/journal_check/bug_refs.json]: Pushed a version with this fixed, thanks

**Comment by simotek** [comment]
I've pushed a new version addressing the previous feedback.

**Pull request merged**